inst/NEWS.Rd

```diff
@@ -41,6 +41,12 @@
     }
     \item Changes in Rcpp Documentation:
     \itemize{
+      \item The Rcpp Quick Reference vignette received a facelift with 
+        new sections on Rcpp attributes and plugins begin added.
+        Furthermore, content was improved in the following
+        sections: important notes, inline compiling, environments,
+        and calling R functions.
```

Comment:
Thanks for adding this.  Four rows is a little longer than what we usually do, that length is better for the Changelog.  So I may trim this.